grafana/base/grafanas/grafana.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  deployment:
+    envFrom:
+      - configMapRef:
+          name: grafana-config-overrides
+  config:
+    server:
+      root_url: $GRAFANA_ROOT_URL
```

I would avoid placeholders that look like variables (because some might be confused about how things operate). Consider something like...

```suggestion
      root_url: REPLACE_IN_OVERLAY
```

---

I have applied this suggestion.